src/qt/askpassphrasedialog.cpp

```diff
@@ -43,10 +43,15 @@ AskPassphraseDialog::AskPassphraseDialog(Mode _mode, QWidget *parent) :
     switch(mode)
     {
         case Encrypt: // Ask passphrase x2
-            ui->warningLabel->setText(tr("Enter the new passphrase to the wallet.<br/>Please use a passphrase of <b>ten or more random characters</b>, or <b>eight or more words</b>."));
+            {
```

Comment:
nit: Why the added block `{ … }`?

Reply:
Recommenting so the edits do not clutter it (specially because I switched "declaration" by "initialization", oops). As you can see [here](http://en.cppreference.com/w/cpp/language/switch): `"Because transfer of control is not permitted to enter the scope of a variable, if a declaration statement is encountered inside the statement, it has to be scoped in its own compound statement:"`. So the alternatives I know are this or declaring the variable before the case and I don't have a preference here; so if this is ugly/confusing, it may be better to always have it declared before the case label.

Reply:
nit: I prefer this inlined too, ala line 120

Reply:
I agree but the line was very convoluted like `tr("Enter the new passphrase.") + <br/> + tr("Please, use......").arg("<b>", "</b>");` in case of avoiding leave the unnecessary `<br/>`'s placeholder (%1) for translators. Tried this  as a middle ground.

Reply:
Meh... thinking it through, there's not enough reason to force this extra block pattern. Much better just breaking the line and using `+` operator, I think. Thx!